resources/views/support/components/payment-grid.blade.php

```diff
@@ -24,12 +24,14 @@
                             </p>
                             @if(!empty($card['cta_options']))
                                 @foreach($card['cta_options'] as $cta_option)
-                                    <a href="{{ $cta_option['cta_link'] }}" aria-describedby="{{ isset($heading_slugified) ? $heading_slugified : '' }}">
-                                        @if(!empty($cta_option['cta_text']))
-                                            {{ $cta_option['cta_text'] }}
-                                        @endif
-                                        @svg('fas-chevron-right', ['width' => '16px', 'height' => '16px', 'color' => '#fff'])
-                                    </a>
+                                    @if(!empty($cta_option['cta_link']))
+                                        <a href="{{ !empty($cta_option['cta_link']) ? $cta_option['cta_link'] : '' }}" aria-describedby="{{ isset($heading_slugified) ? $heading_slugified : '' }}">
```

Comment:
The `!empty($cta_option['cta_link'])` is redundant - you are already checking `!empty($cta_option['cta_link'])` on line 27.

I would recommend removing this check to simplify the code.